demisto_sdk/commands/common/errors.py

```diff
@@ -2172,3 +2176,15 @@ def aliases_with_inner_alias(invalid_aliases: List[str]):
         return "The following fields exist as aliases and therefore cannot contain an 'Aliases' key."\
                f"\n{invalid_aliases}\n" \
                "Please remove the key from the fields or removed the fields from the other field's Aliases list."
+
+    @staticmethod
+    @error_code_decorator
+    def missing_alternative_fields_nested_items():
+        return 'Alternative fields of nested items were found and are missing from this file, ' \
+               'please add them to the file.'  # TODO: improve msg
+
+    @staticmethod
+    @error_code_decorator
+    def missing_alternative_fields_in_using_items(using_id, item_id):
+        return f'Item with id {using_id} is using the item with id {item_id} and it is missing its alternative ' \
+               f'fields. Please add the missing alternative fields to the file.'  # TODO: improve msg
```

Comment:
@yaakovi could use your input with a better error msg in these too errors.
plus - this will suggest a solution for running format on each of the 'using' files (meaning many different calls of format). this is the desired behavior, right? or do we want to allow format to automatically run on all of the using files when running on the single file? (the one that have the alternative field)